docs/codeql/support/reusables/versions-compilers.rst

```diff
@@ -4,9 +4,9 @@
    :stub-columns: 1
 
    Language,Variants,Compilers,Extensions
-   C/C++,"C89, C99, C11, C18, C++98, C++03, C++11, C++14, C++17","Clang (and clang-cl [1]_) extensions (up to Clang 9.0),
+   C/C++,"C89, C99, C11, C18, C++98, C++03, C++11, C++14, C++17","Clang (and clang-cl [1]_) extensions (up to Clang 12.0),
```

Comment:
@criemen and @AlexDenisov, is at least one of you comfortable with us claiming support for Clang 12.0? I'm guessing your laptops is the only place where we test it.

Reply:
@jbj yes, I think we are good to go.

Reply:
I agree - I've not seen any breakage, so we can claim support, and we'd most certainly want to hear from customers if clang 12 breaks.